packages/grid/x-data-grid/src/colDef/gridBooleanOperators.ts

```diff
@@ -5,13 +5,13 @@ import { GridFilterOperator } from '../models/gridFilterOperator';
 export const getGridBooleanOperators = (): GridFilterOperator<any, boolean | null, any>[] => [
   {
     value: 'is',
-    getApplyFilterFn: (filterItem: GridFilterItem) => {
+    getApplyFilterFnV7: (filterItem: GridFilterItem) => {
       if (!filterItem.value) {
         return null;
       }
 
       const valueAsBoolean = filterItem.value === 'true';
-      return ({ value }): boolean => {
+      return (value, _, __, ___): boolean => {
```

Comment:
Why not skip the unused arguments?

Reply:
Old habit, V8 [used to have](https://v8.dev/blog/adaptor-frame) an adaptor for functions with mismatched arity that was expensive, but the overhead is mostly gone now. But I still like to write javascript code that is easy & predictable for engines to optimize. In particular for functions like this one that are run in a hot loop.

Reply:
Thanks for the article, I have learned a lot about v8 lately 🎉

Reply:
Np, I love learning about JS engines internals, helps a lot with performance optimization. I'm less familiar with SpiderMonkey though, and I know very little about JSC. But V8 is the most common engine by far.

If you're interested in reading more, those links are all interesting:
 - https://v8.dev/blog
 - https://github.com/thlorenz/v8-perf
 - https://mrale.ph/